pages/common/if.md

```diff
@@ -1,27 +1,28 @@
 # if
 
 > Simple shell conditional.
+> See also: `test`, `[`.
 
-- Echo a different thing depending on a command's success:
+- Executed two different commands based on a condition:
 
-`if {{command}}; then echo "success"; else echo "failure"; fi`
```

Comment:
I would keep this, to show that if can be used with any command, not only with `[` or `test`...

Reply:
Well, I removed it because it's nearly a duplicate of the next example :/

Reply:
Then maybe just keep this one. `if` does not run on a "condition", but rather on any command (built-in or not)...

Reply:
Ah good point.  I should have tested that 😄

Reply:
Fixed https://github.com/tldr-pages/tldr/pull/5351/commits/c46b7b93d6fcd420e1ce6670bc4d07ebe3865f54